src/main/java/org/gradle/profiler/mutations/ClearArtifactTransformCacheMutator.java

```diff
@@ -17,15 +13,7 @@ public ClearArtifactTransformCacheMutator(File gradleUserHome, CleanupSchedule s
 
     @Override
     protected void cleanupCacheDir(File cacheDir) {
-        filesAsStream(
-            cacheDir,
-            (dir, name) -> name.startsWith("files-")
-        ).flatMap(file -> filesAsStream(file, (dir, name) -> !name.endsWith(".lock")))
-            .forEach(AbstractCleanupMutator::delete);
-    }
-
-    private Stream<File> filesAsStream(File dir, FilenameFilter filter) {
-        return Arrays.stream(Objects.requireNonNull(dir.listFiles(filter)));
+        delete(cacheDir);
```

Comment:
Can we delete the cache dir when the daemon is running? I suppose we won't be able to remove the `.lock` file, right?